Added 8xlarge and 16xlarge sizes of m5, m5a, m5d, r5, r5a, and r5d AWS instance types - #52667(bugtrack)-trunk

abika5 opened the pull request.

abika5 committed:
Added 8xlarge and 16xlarge sizes of m5, m5a, m5d, r5, r5a, and r5d AWS instance types - #52667(bugtrack)-trunk

mattsunsjf reviewed (commented):
@abika5  Are these changes going into MarkLogic 11 only? Should there be a PR going into 10.0-develop?

abika5 commented:
@mattsunsjf I have added these changes or 10-develop too (https://github.com/marklogic/cloud-enablement-aws/pull/107) and PR was created. This PR had 2 commits I think after you have reviewed one of the commits, I added another commit and merged the PR. Sorry, my bad I didn't inform you.

mattsunsjf commented:
> @mattsunsjf I have added these changes or 10-develop too (#107) and PR was created. This PR had 2 commits I think after you have reviewed one of the commits, I added another commit and merged the PR. Sorry, my bad I didn't inform you.

Hi @abika5 , there should be no more changes after a pull request is reviewed. Otherwise, it should go over the review again.
